packages/gatsby-plugin-google-analytics/index.d.ts

```diff
@@ -0,0 +1,8 @@
+  import * as React from "react";
+
+  interface OutboundLinkProps {
+    onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
```

Comment:
I omitted `href` and `target` from these props because I'm extending the default `HTMLAnchorElement` props, which include those